Sources/Plasma/CoreLib/HeadSpin.h

```diff
@@ -380,7 +380,9 @@ constexpr float hsInvert(float a) { return 1.f / a; }
 #ifdef _MSC_VER
 #   define ALIGN(n) __declspec(align(n))
 #else
-#   define ALIGN(n) __attribute__((aligned(n)))
+#   ifndef ALIGN
+#       define ALIGN(n) __attribute__((aligned(n)))
+#   endif
```

Comment:
Only my Mac it's saying ALIGN is already defined. Only redefining it if it isn't defined already.

Reply:
This seems like it's going to be a problem... the existing `ALIGN` macro in param.h on macOS does not set the alignment attribute...

```cpp
// param.h
#define ALIGN(p)        __DARWIN_ALIGN(p)
```

```cpp
// _param.h
#define __DARWIN_ALIGNBYTES     (sizeof(__darwin_size_t) - 1)
#define __DARWIN_ALIGN(p)       ((__darwin_size_t)((__darwin_size_t)(p) + __DARWIN_ALIGNBYTES) &~ __DARWIN_ALIGNBYTES)
```

We probably (in another PR) want to rename `ALIGN` to something like `HS_ALIGNED`

Reply:
Wouldn't it be better to switch to C++11 standard [`alignas`](https://en.cppreference.com/w/cpp/language/alignas) instead?

Reply:
yeah, that sounds like a good solution :+1:

Reply:
Is the ALIGN macro even used anywhere? I did a quick text search but couldn't find anything.

Reply:
I'm just looking at this now, and it looks like the only user is plDXPipeline for setting up SSE skinning. So that code maybe got copied into plGLPipeline and plMetalPipeline, but should be easy enough to clean up

Reply:
Metal is using Apple's Accelerate library for that math, so that wouldn't have impacted Mac. But that explains why my Xcode project isn't seeing it.